app/buyers/views/buyers.py

```diff
@@ -0,0 +1,282 @@
+        )
+    except HTTPError as e:
+        update_data = section.unformat_data(update_data)
+        errors = section.get_error_messages(e.message, lot_slug)
+
+        return render_template(
+            "buyers/edit_brief_section.html",
+            framework=framework,
+            data=update_data,
+            section=section,
+            errors=errors,
+            **dict(buyers.config['BASE_TEMPLATE_DATA'])
+        ), 200
+
+    return redirect(
+        url_for(".view_brief_summary", framework_slug=framework_slug, lot_slug=lot_slug, brief_id=brief_id)
+    )
+
+
+@buyers.route('/buyers/frameworks/<framework_slug>/requirements/<lot_slug>/<brief_id>', methods=['GET'])
+def view_brief_summary(framework_slug, lot_slug, brief_id):
+    framework, lot = get_framework_and_lot(framework_slug, lot_slug, data_api_client)
+
+    if framework['status'] != 'live':
+        abort(404)
+    if not lot['allowsBrief']:
+        abort(404)
+
+    brief = data_api_client.get_brief(brief_id)["briefs"]
+    if not is_brief_associated_with_user(brief, current_user.id):
+        abort(404)
+
+    content = content_loader.get_manifest(framework_slug, 'edit_brief').filter(
+        {'lot': lot['slug']}
+    )
+    sections = content.summary(brief)
+    unanswered_required, unanswered_optional = count_unanswered_questions(sections)
+    delete_requested = True if request.args.get('delete_requested') else False
+
+    # TODO: check validation errors(?)
+    validation_errors = None
+
+    flattened_brief = []
+    for section in sections:
+        for question in section.questions:
+            question.section_id = section.id
+            flattened_brief.append(question)
+
+    return render_template(
+        "buyers/brief_summary.html",
+        framework=framework,
+        confirm_remove=request.args.get("confirm_remove", None),
+        brief_id=brief_id,
+        brief_data=brief,
+        last_edit=brief['updatedAt'],
+        flattened_brief=flattened_brief,
+        unanswered_required=unanswered_required,
+        unanswered_optional=unanswered_optional,
+        can_publish=not validation_errors and not unanswered_required,
+        delete_requested=delete_requested,
+        **dict(buyers.config['BASE_TEMPLATE_DATA'])
+    ), 200
+
+
+@buyers.route('/buyers/frameworks/<framework_slug>/requirements/<lot_slug>/<brief_id>/delete', methods=['POST'])
+def delete_a_brief(framework_slug, lot_slug, brief_id):
+    framework, lot = get_framework_and_lot(framework_slug, lot_slug, data_api_client)
+
+    if framework['status'] != 'live':
+        abort(404)
+    if not lot['allowsBrief']:
+        abort(404)
+
+    brief = data_api_client.get_brief(brief_id)["briefs"]
+    if not is_brief_associated_with_user(brief, current_user.id):
+        abort(404)
+
+    if request.form.get('delete_confirmed'):
+        # TODO: Delete the brief (once a DELETE endpoint exists in the API)
+        flash({"requirements_deleted": brief.get("title")})
```

Comment:
Are we allowing the deletion of briefs? If so, this should probably raise a `NotImplementedError`.